src/Text.jsx

```diff
@@ -5,38 +5,36 @@
  * Given:
  * <Localization messageBundle={{ strings: { HelloWorld: 'Hello, World!' }}}
  *  <AnyParent>
- *    <Text key='localized.strings.HelloWord' style={{ color: 'blue' }} />
+ *    <Text message='localized.strings.HelloWord' style={{ color: 'blue' }} />
  *  </AnyParent>
  * </Localization>
  *
  * Output: <span style={{ color: 'blue' }}>Hello, World!</span>
  */
 
 import React, { PropTypes } from 'react';
-const { array, func, string } = PropTypes;
+const { array, bool, func, string } = PropTypes;
 
 const Text = (props, context) => {
-  const { message, children, values, ...rest } = props;
-  const { localize } = context;
+  const { message, values, ...rest } = props;
+  const { localize, _localizeDebug } = context;
 
   let localized = message;
 
   if (typeof localize === 'function') {
-    localized = localize(message, ...values);
+    localized = localize(message, [...values]);
   }
 
-  // allow for optional alternative API using render-prop pattern
-  if (typeof children === 'function') {
-    return children(localized, message, ...values);
-  } else {
-    return (<span {...rest}>{localized}</span>);
+  if (_localizeDebug) {
+    rest['data-original-message'] = message;
   }
+
+  return (<span {...rest}>{localized}</span>);
```

Comment:
Just curious: In practice, have you ever been bitten by *not* using the parens around JSX?